src/index.js

```diff
@@ -1,20 +1,16 @@
 import readlineSync from 'readline-sync';
-import getRandomNumber from './randomCount.js';
 
 const numberOfRound = 3;
-const brainEven = () => {
+const theCoreOfGames = (brainRulls) => {
```

Comment:
Plz don't use 'the' in var names. Maybe gamesCore or gameEngine?

Reply:
brainRulls unclear name for the passed function

Reply:
fixed